src/components/Header.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import './Header.css';
+
+const Header = () => (
+    <>
+      <header className="headsection">
+        <div className="inner-head">
+          <span className="Bookstore-CMS Text-Style-5">
+            Bookstore CMS
+          </span>
+          <nav className="lists">
+            <a href="/Books" className="BOOKS Text-Style-3">
+               BOOKS
+            </a>
+            <a href="/categories" className="CATEGORIES Text-Style-3">
+               CATEGORIES
+            </a>
+          </nav>
```

Comment:
- [ ] Kindly use `<link>`s instead of using `<a>` elements, as per [this](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_init_project_wth_comps_v2_0.md#:~:text=%3CRoute%3Es%20and%20%3CLink%3Es) requirement.